src/Games/NexusMods.Games.StardewValley/Installers/SMAPIInstaller.cs

```diff
@@ -226,17 +229,10 @@ await using (var stream = await item.OpenAsync())
                 Name = "SMAPI",
                 Id = info.BaseModId,
                 Files = modFiles,
-                SortRules = new[]
+                Metadata = new TempEntity
                 {
-                    new SMAPISorter(),
+                    {SMAPIMarker.Version, version},
                 },
```

Comment:
Do we deploy SMAPI correctly without this? I had to explicitly add this sorted because sorting is otherwise undefined and we wouldn't be able to override the game executable.